### PR TITLE
rTorrent: Help menu version and bugs link

### PR DESCRIPTION
**Pull request by ac1dburnz**
Just a general idea of how to make Changes to menu to determine version which are updated every release

### CODE REVIEW

File: rtorrent/src/main.cc [hunk 1]
```diff
@@ -647,7 +647,7 @@ do_panic(int signum) {
 
 void
 print_help() {
-  std::cout << "Rakshasa's BitTorrent client version " VERSION "." << std::endl;
+  std::cout << "This is stickz modified version 0.9.8-v5.0 of Rakshasa's BitTorrent client version " VERSION "." << std::endl;
```
Comment (stickz) [inline rtorrent/src/main.cc]: Change to this:
`std::cout << "rtorrent stickz BitTorrent client version " STICKZ_VERSION "." << std::endl;`

Change this to link to the GitHub issues report section.
https://github.com/stickz/rtorrent/blob/6598f49fd548acaf44aaf102c1564c104b4c2e02/rtorrent/src/main.cc#L689

Add on Line 4: `AC_DEFINE(STICKZ_VERSION, "0.9.8-v5.0", rtorrent stickz version)`
https://github.com/stickz/rtorrent/blob/6598f49fd548acaf44aaf102c1564c104b4c2e02/rtorrent/configure.ac#L3-L4

**Review by stickz — CHANGES_REQUESTED** [review]

**Comment by stickz** [comment]
We're not changing the version we announce to peers in the BitTorrent stream yet. We need to coordinate this carefully.

**Comment by ac1dburnz** [comment]
lmk if there is anything else.

File: rtorrent/src/main.cc [hunk 2]
```diff
@@ -686,7 +686,8 @@ print_help() {
   std::cout << "  o                 View trackers" << std::endl;
   std::cout << std::endl;
 
-  std::cout << "Report bugs to <sundell.software@gmail.com>." << std::endl;
+   std::cout << "Report bugs to <https://github.com/stickz/rtorrent/issues/new?assignees=&labels=&projects=&template=bug_report.yml>." << std::endl; 
+
```
Comment (stickz) [inline rtorrent/src/main.cc]: @ac1dburnz  **This is enough:** https://github.com/stickz/rtorrent/issues

**Review by stickz — CHANGES_REQUESTED** [review]

**Pull request merged**